Source/ObjectiveDropboxOfficial/Shared/Handwritten/OAuth/DBOAuthManager.h

```diff
@@ -28,15 +29,30 @@ NS_ASSUME_NONNULL_BEGIN
 /// the `team_id` if (team app).
 @property (nonatomic, readonly, copy) NSString *uid;
 
+/// The refresh token if accessToken is short-lived.
+@property (nonatomic, readonly, copy) NSString *refreshToken;
+
+/// The expiration time of the (short-lived) accessToken.
+@property (nonatomic, readonly, assign) NSTimeInterval tokenExpirationTimestamp;
+
+/// Convenience method for initWithAccessToken:uid:refreshToken:tokenExpirationTimestamp: with
+/// refreshToken set to nil and tokenExpirationTimestamp set to 0.
+- (instancetype)initWithAccessToken:(NSString *)accessToken uid:(NSString *)uid;
```

Comment:
Good point, I will add two factory methods and keep the existing interfaces so that it continues to work for developer who are using the old interface.